### PR TITLE
Update Python.gitignore

### PR DESCRIPTION
**Pull request by wind-zrf**
add pyc

### CODE REVIEW

File: Python.gitignore [hunk 1]
```diff
@@ -61,6 +61,7 @@ cover/
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
+*.pyc
```
Comment (shiftkey) [inline Python.gitignore]: This should be covered by line 3

https://github.com/github/gitignore/blob/8170c0ed25ecf564389332896f44b5aec359282f/Python.gitignore#L3

**Comment by shiftkey** [comment]
@wind-zrf can you confirm that the existing rule works for you?

File: Python.gitignore (hunk 1 above)
Comment (michaelotty) [reply Python.gitignore]: Agreed, this is covered in line 3 according to the git docs shown below, and works for me at ignoring .pyc files.

> [An asterisk "*" matches anything except a slash. The character "?" matches any one character except "/". The range notation, e.g. [a-zA-Z], can be used to match one of the characters in a range. See fnmatch(3) and the FNM_PATHNAME flag for a more detailed description.](https://git-scm.com/docs/gitignore)

**Comment by pradyunsg** [comment]
I'm pretty sure this can be closed. :)

**Comment by shiftkey** [comment]
Closing this out due to inactivity and unresolved questions.

**Pull request closed without merging**